src/app/area/areas-arborescence/areas-arborescence.page.html

```diff
@@ -13,7 +13,7 @@
 
 
       <ion-buttons slot="end">
-        <ion-button  (clic)="clicValid()" size="large">
+        <ion-button  (clic) ="optionsClicked()" size="large">
```

Comment:
`(clic)` is not a valid Angular event. `(click)` is the valid DOM event that you should use here.
Also, please remove the space that you added before the `=`